lib/tasks/poll.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+namespace :poll do
+  desc "set status expired polls to closed"
+  task finish_expired: :environment do
```

Comment:
If this your primary way of closing expired polls, I would recommend you to think again — what will happen if your cron job will fail because of some syntax error? My primary approach would be `after_find` ActiveRecord callback changing state attribute (and I wouldn't even bother saving this state back to DB, it will happen automatically if we save the object later — or not happen at all if we're read-only, which is also OK by me.)